Cargo.toml

```diff
@@ -5,16 +5,12 @@ authors = [ "Robert Clipsham <robert@octarineparrot.com>" ]
 license = "MIT/Apache-2.0"
 homepage = "https://github.com/libpnet/libpnet"
 repository = "https://github.com/libpnet/libpnet"
-documentation = "http://octarineparrot.com/assets/libpnet/doc/pnet/"
 description = "Cross-platform, low level networking using the Rust programming language."
 readme = "README.md"
 keywords = ["networking", "transport", "datalink", "packet", "protocol"]
 categories = ["network-programming", "parser-implementations"]
 build = "build.rs"
 
-[lib]
-name = "pnet"
```

Comment:
Redundant since by default the lib name is the same as the package name.

Reply:
Interesting historical point: I actually wanted `pnet` to be `libpnet`, but it was too late once it was published.

Reply:
May I ask why? What is the difference between this crate and all other library crates not named `lib...`?

Reply:
Much like `libpcap` is not just `pcap`, `libc` is not just `c`, `libpnet` is not just `pnet`. Except when it is now... 😛